syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -43,11 +43,17 @@
 				{
 					"name": "keyword.control.scl",
 					"match": "(?i)\\b(and|or|xor|not)\\b"
+				},
+				{
+					"name": "constant.regexp.scl",
+					"match": "\\#.\\w*(?!\\()\\b[.]\\w.*?\\b|\\#\\D.\\w*(?!\\()\\b|\\.\\w*|\\.\\w*"
+				},
+				{
+					"name": "constant.regexp.scl",
+					"match": "(^ (\\s*\\w*?) (?=:))|(^(\\s*\\w*?) )(?={)"
```

Comment:
What does this do?

Reply:
This one will highlight the variable declaration: https://regex101.com/r/uQ7IgK/1

Reply:
nice. Same as above. Maybe we should change the name

Reply:
Updated.